demos/mcmcp/experiment.py

```diff
@@ -56,6 +56,11 @@ def add_node_to_network(self, node, network):
         parent = node.neighbors(direction="from")[0]
         parent.transmit()
         node.receive()
+        
+    def data_check(self, participant):
+        """ Make sure each trial contains exactly one chosen info """
```

Comment:
We have a [PEP8](https://www.python.org/dev/peps/pep-0008/) linter running over PRs and it is complaining that you didn't format the docstring like this, removing the surrounding spaces and adding a period at the end:

```
"""Make sure each trial contains exactly one chosen info."""
```